R/ospsuite.utils-env.R

```diff
@@ -0,0 +1,45 @@
+# Environment that holds various global variables and settings for the ospsuite.utils,
+# It is not exported and should not be directly manipulated by other packages.
+ospsuiteUtilsEnv <- new.env(parent = emptyenv())
+
+# name of the package. This will be used to retrieve information on the package at run time
+ospsuiteUtilsEnv$packageName <- "ospsuite.utils"
+
+ospsuiteUtilsEnv$suiteName <- "Open Systems Pharmacology"
+
+# Separator defined in OSPSuite.Core.
+ospsuiteUtilsEnv$pathSeparator <- "|"
+
+# Default values for the formatNumerics() helper function
+ospsuiteUtilsEnv$formatNumericsDigits <- 2
+
+# Specifies the symbol used for µ. This will be set by the .NET layer
+ospsuiteUtilsEnv$muSymbol <- "µ"
```

Comment:
Warning during build - is this a problem?

Reply:
Nothing relies here on .NET code, it's just a symbol for "micro" that should be consistently used throughout the code in all packages. For whatever reasons, there are different "micro" symbols (crazy).

Reply:
It's fine to leave this settings in ospsuite-r. It is used as a default value but is set at run time from the .net engine. So let's remove it. 

I would not define settings here that are not used in the package. Otherwise this will become just a collection of settings where usage will be quite hard to track

Reply:
@msevestre could you point me where it is set from .net engine? I see it being defined only once.

Reply:
You know what. Maybe we realized it's not needed anymore as long as we set it using the right encoding in R.....I know we went back and forth with it but I was wrong. We just used it now as the "right" constant...

Reply:
This is back in ospsuite.